repo/packages/C/crate/0/marathon.json

```diff
@@ -0,0 +1,26 @@
+{
+    "id": "/crate/{{crate/cluster-name}}",
+    "instances": 1,
+    "cpus": 0.25,
+    "mem": 128,
+    "ports": [{{framework/api-port}}],
+    "env": {
+{{#framework/auth/enabled}}
+      "MESOS_AUTHENTICATE": "true",
+      "DEFAULT_PRINCIAL": "{{framework/auth/princial}}",
+      "DEFAULT_SECRET": "{{framework/auth/secret}}"
+{{/framework/auth/enabled}}
+    },
+    "uris": [
+        "https://downloads.mesosphere.io/java/jre-7u76-linux-x64.tar.gz",
+        "https://cdn.crate.io/downloads/releases/crate-mesos-0.1.0-SNAPSHOT-cec0ad0.tar.gz"
```

Comment:
Please see http://www.oracle.com/technetwork/java/javase/readme-142177.html#redistribution for a full outline of licensing information about redistribution of the oracle jre.